README.md

```diff
@@ -188,6 +188,7 @@ This example shows [all the possible configuration items of ngrok.forward](https
 ```python
 listener = ngrok.forward(
     # session configuration
+    app_protocol="http2",
```

Comment:
nit: could this go below the authtoken bits, since addr and authtoken are the most important bits for folks to get right?